modules/ROOT/pages/upgrade.adoc

```diff
@@ -3,89 +3,9 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-== Before You Begin
+== MuleSoft Professional Services Required
+MuleSoft professional services must be involved throughout the upgrade process. 
+
 Because Anypoint Platform Private Cloud Edition (Anypoint Platform PCE) upgrades require a working installation, all environment prerequisites should already be met. As long as you (or your infrastructure team) successfully performed all prerequisite tasks listed in xref:prereq-hardware.adoc[Hardware Prerequisites], xref:prereq-software.adoc[Software Prerequisites], and xref:prereq-network.adoc[Network Prerequisites] prior to the installation of Anypoint Platform PCE, your environment can be upgraded successfully.
 
 Refer to the xref:release-notes::pce/anypoint-private-cloud-release-notes.adoc[Anypoint Platform Private Cloud Edition Release Notes] for additional information about upgrading Anypoint Platform PCE to the most recent version.
```

Comment:
I think in the release notes we removed the additional information about upgrading PCE. What about changing this line to:

Refer to the xref:release-notes::pce/anypoint-private-cloud-release-notes.adoc[Anypoint Platform Private Cloud Edition Release Notes] for additional information about the most recent Anypoint Platform PCE version.

Reply:
Makes sense - just updated.